www/index.html

```diff
@@ -172,6 +172,9 @@ <h3>Code Plugins</h3>
             <li><a href="https://github.com/thejohnhoffer/viaWebGL">OpenSeadragonGL</a> allows you to run WebGL shaders on all tiles in OpenSeadragon.</li>
             <li><a href="https://github.com/KTGLeiden/Openseadragon-screenshot">OpenSeadragonScreenshot</a> allows you to make a screenshot of your viewport, with optional magnification.</li>
             <li><a href="https://github.com/picturae/OpenSeadragonGuides">OpenSeadragonGuides</a> allows you to add horizontal and vertical guidelines to the Openseadragon viewer.</li>
+            <li><a href="https://github.com/thejohnhoffer/viaWebGL">GLSL Shaders on OpenSeadragon</a> allows you to add shaders to zoomable tiles or any other image viaWebGL.</li>
```

Comment:
Looks like this is already in the list (see line 172). Sorry for the state of our bug list!